tests/ballerina-compiler-api-test/src/test/resources/test-src/expected-type/annotation_test.bal

```diff
@@ -0,0 +1,297 @@
+type Info record {|
+    int id;
+    string name?;
+    anydata description;
+|};
+
+type AnnotationConfig record {|
+    Info info;
+    int|record {*Info; string description;} doc?
+|};
+
+
+const annotation AnnotationConfig CustomAnnotation on source type, source const, source class,
+                                                        source function, source object function,
+                                                        source service, source return, source parameter,
+                                                        source field, source listener, source var,
+                                                        source annotation, source worker;
+
+@CustomAnnotation {
+    info: {},
+    doc: {}
+}
+type TestString string;
+
+@CustomAnnotation {
+    info: {},
+    doc: {}
+}
+enum TestEnum {
+    FIRST, SECOND
+}
+
+@CustomAnnotation {
+    info: {},
+    doc: {}
+}
+const int TEST_CONST = 32;
+
+@CustomAnnotation {
+    info: {},
+    doc: {}
+}
+function testFunction() {
+
+}
+
+@CustomAnnotation {
+    info: {},
+    doc: {}
+}
+isolated class TestClass {
+    @CustomAnnotation {
+        info: {},
+        doc: {}
+    }
+    int i;
+
+    @CustomAnnotation {
+        info: {},
+        doc: {}
+    }
+    function testClassFunction() {
+
+    }
+
+    public function init() {
+    }
+
+    public function attach(service object {} s, string[]|string|() name = ()) returns error? {
+    }
+
+    public function 'start() returns error? {
+    }
+
+    public function gracefulStop() returns error? {
+    }
+
+    public function immediateStop() returns error? {
+    }
+
+    public function detach(service object {} s) returns error? {
+    }
+}
+
+@CustomAnnotation {
+    info: {},
+    doc: {}
+}
+type TestObject object {
+    @CustomAnnotation {
+        info: {},
+        doc: {}
+    }
+    int i;
+
+    @CustomAnnotation {
+        info: {},
+        doc: {}
+    }
+    function testObjectFunction();
+};
+
+@CustomAnnotation {
+    info: {},
+    doc: {}
+}
+service / on new TestClass() {
+    @CustomAnnotation {
+        info: {},
+        doc: {}
+    }
+    remote function name() {
+
+    }
```

Comment:
```suggestion
    remote function name() {
    }
```

If we don't want a new line within the function body block, we can remove it. Please check other places too